src/System.Net.Security/src/System/Net/SecurityStatusAdapterPal.Windows.cs

```diff
@@ -70,38 +58,26 @@ internal static SecurityStatusPal GetSecurityStatusPalFromNativeInt(int win32Sec
 
         internal static SecurityStatusPal GetSecurityStatusPalFromInterop(Interop.SecurityStatus win32SecurityStatus)
         {
-            foreach (SecurityStatusMapping mapping in s_securityStatusMapping)
+            SecurityStatusPal status;
+            bool ret = StatusDictionary.TryGetForward(win32SecurityStatus, out status);
+            if (!ret)
```

Comment:
Nit: you don't need the temporary bool... this can just be:
```C#
if (!StatusDictionary.TryGetForward(win32SecurityStatus, out status))
{
    ...
}
```